index.html

```diff
@@ -1,53 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <!-- Required meta tags -->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
-  <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-  
-  
-  <!-- Bootstrap CSS -->
-  <link href="css/bootstrap.css" rel="stylesheet">
-  
-  <!-- Bootstrap Font Icon CSS -->
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
-  
-  <!-- Custom CSS -->
-  <link href="css/style.css" rel="stylesheet">
-<link rel="stylesheet" href="css/style.css">
-  <title>Frontend Mentor | NFT preview card component</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- displays site properly based on user's device -->
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
+
+
+    <!-- Bootstrap CSS -->
+    <link href="css/bootstrap.css" rel="stylesheet">
+
+    <!-- Bootstrap Font Icon CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+
+    <!-- Custom CSS -->
+    <link href="css/style.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Frontend Mentor | NFT preview card component</title>
 
 </head>
+
 <body>
-  <main>
-    <div class="card">
-        <div class="container">
-          <a href="#">
-            <img class="card-persona" src="images/image-equilibrium.jpg" alt="image of equilibrium">
-          </a>
-          <div class="overlay">
-            <img src="images/icon-view.svg" alt="Eye Icon">
-          </div>
-        </div>
-        <h1>Equilibrium #3429</h1>
-        <p>Our Equilibrium collection promotes balance and calm.</p>
-        <div class="cta">
-          <img src="images/icon-ethereum.svg" alt="" aria-hidden="true">
-          <p>0.041 ETH</p>
-          <img class="clock" src="images/icon-clock.svg" alt="" aria-hidden="true">
-          <p>3 days left</p>
+    <main>
+        <div class="card">
+            <div class="container">
+                <a href="#">
+                    <img class="card-persona" src="images/image-equilibrium.jpg" alt="image of equilibrium">
+                </a>
+                <div class="overlay"></div>
```

Comment:
@kmeganiz I split the overlay and icon as a separate elements because you are aplying opacity on a parent and that attribute is going to affect children

Reply:
Ok. Thank you. So overlay on the top of the layer?